arango/request.py

```diff
@@ -2,13 +2,21 @@
 
 from typing import Any, MutableMapping, Optional
 
-from arango.typings import Fields, Headers, Params
+from arango.typings import DriverFlags, Fields, Headers, Params
 
 
-def normalize_headers(headers: Optional[Headers]) -> Headers:
+def normalize_headers(
+    headers: Optional[Headers], driverFlags: Optional[DriverFlags] = None
```

Comment:
nit: `driverFlags` -> `driver_flags` (we won't format the field names in JSON payload and response bodies to be snake_case anymore, but we should still use snake_case for the Python code itself).

Reply:
This is fixed.